packages/ui-components/src/components/text-field/text-field.types.ts

```diff
@@ -0,0 +1,20 @@
+export enum EValidationState {
```

Comment:
I think that the interfaces that need be exposed to the outside need to be manually added to React
at `packages/react-ui-components/src/components/index.ts`. @miguelpinto98 is correct?

Reply:
Not currently. We change this recently, to export all from ui-components.

Reply:
not needed anymore